code/src/components/MessageInput.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import './messageInput.css'
+
+export const MessageInput = () => {
+  const MESSAGES_URL = "https://technigo-thoughts.herokuapp.com";
+  const [message, setMessage] = useState("");
+
+  const handleClick = event => {
+    event.preventDefault();
+
+    fetch(MESSAGES_URL, 
+      {
+        method: "POST",
+        headers: {
+          'Content-Type':'application/json'
+        },
+        body:JSON.stringify({message})
+      }
+    ).then(()=>{
+      window.location.reload();
+    })
+  };
+  
+  return (
+    <section className="input-section">
+      <form className="message-card">
+        <p>What's making you happy right now?</p>
+        <textarea className="textarea"
+          rows= '2'
+          type="text"
```

Comment:
Here you use both `'` and `"`. Would look nicer if you choose one way of writing!

Reply:
Absolutely, big thanks!